extensions/amp-empower/OWNERS.yaml

```diff
@@ -0,0 +1,5 @@
+# For an explanation of the OWNERS.yaml rules and syntax, see:
+# https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example.yaml
+
+- ampproject/wg-ui-and-a11y
+- merty
```

Comment:
Please use the JSON format for all new OWNERS files. This will become:

```javascript
// For an explanation of the OWNERS rules and syntax, see:
// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example

{
  rules: [
    {
      owners: [
        { name: "ampproject/wg-ui-and-a11y" },
        { name: "merty" }
      ]
    }
  ]
}
```

Reply:
Done 👍